p2panda-net/src/config.rs

```diff
@@ -10,25 +12,30 @@ use crate::{NetworkId, NodeAddress, RelayUrl};
 pub const DEFAULT_BIND_PORT: u16 = 2022;
 
 /// Default network id.
-pub const DEFAULT_NETWORK_ID: NetworkId = [0; 32];
+pub const DEFAULT_NETWORK_ID: NetworkId = [
+    247, 69, 248, 242, 132, 120, 159, 230, 98, 100, 214, 200, 78, 40, 79, 94, 174, 8, 12, 27, 84,
+    195, 246, 159, 132, 240, 79, 208, 1, 43, 132, 118,
+];
```

Comment:
Thought it would be cool to have a decent default. This is a BLAKE3 hash over the string "p2panda" ❤️